gradle-plugin/doc/src/main/asciidoc/inc/generator/_overview.adoc

```diff
@@ -2,7 +2,8 @@ The usual way to define Docker images is with the plugin configuration as explai
 
 However, this plugin provides an additional route for defining image configurations. This is done by so called _Generators_. A generator is a Java component providing an auto-detection mechanism for certain build types like a Spring Boot build or a plain Java build. As soon as a _Generator_ detects that it is applicable it will be called with the list of images configured in the `build.gradle`. Typically a generator only creates dynamically a new image configuration if this list is empty. But a generator is free to also add new images to an existing list or even change the current image list.
```

Comment:
I don't understand the 2 last sentences. Maybe worth removing them if it is confusing and do not provide any value to the user.
But this could be improved in another PR